week2/homework/src/add.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+const fs = require('fs');
+
+function add(item) {
```

Comment:
great job to split this up into separate function and in separate files 👍

Reply:
dear @gajduk thank you so much for your super nice feedback.

Reply:
For the HW3, I will try to implement codes with promises .